cuqi/samples/_samples.py

```diff
@@ -325,7 +347,7 @@ def plot(self,sample_indices=None,*args,**kwargs):
 
 
     def plot_chain(self, variable_indices=None, *args, **kwargs):
-        dim = self.geometry.dim
```

Comment:
I think this will cause issues for function value Samples objects that are multi dimensional as is the case for e.g. `Image2D`. Perhaps plot_chain should not work for funvals samples arrays.

Reply:
Great catch! Now `funvals` only works for geometries with functions represented by 1D ndarray